### ISSUE TITLE
 train.py

### ISSUE THREAD

**Issue by mymailpath** [ldulcic/chatbot#2]
Need solution for this stage, because it took long time to go further.,,
and dont know how to view .pt files
///
connected to pydev debugger (...,...,...,)
Namespace(attention_score='dot', attention_type='global', batch_size=32, concat_attention_hidden_size=128, cuda=False, dataset='twitter-applesupport', decoder_hidden_size=128, decoder_init_type='zeros', decoder_num_layers=1, decoder_rnn_cell='LSTM', decoder_rnn_dropout=0.2, decoder_type='bahdanau', embedding_size=25, embedding_type='glove.twitter.27B.25d', encoder_bidirectional=False, encoder_hidden_size=128, encoder_num_layers=1, encoder_rnn_cell='LSTM', encoder_rnn_dropout=0.2, gradient_clip=5, half_window_size=5, learning_rate=0.0001, local_p_hidden_size=128, luong_attn_hidden_size=128, luong_input_feed=False, max_epochs=1, multi_gpu=False, save_every_epoch=False, save_path='/home/administrator/Downloads/chatbot-develop/result/2019-05-14-16:01', teacher_forcing_ratio=0.5, train_embeddings=False)
Using CPU for training
Loading dataset...Done.
Saving vocab and args...Done
/home/administrator/Documents/setup/lib/python3.5/site-packages/torch/nn/modules/rnn.py:38: UserWarning: dropout option adds dropout after all but last recurrent layer, so non-zero dropout expects num_layers greater than 1, but got dropout=0.2 and num_layers=1
  "num_layers={}".format(dropout, num_layers))
Seq2SeqTrain(
  (encoder): SimpleEncoder(
    (embed): Embedding(12355, 25, padding_idx=1)
    (rnn): RNNWrapper(
      (rnn): LSTM(25, 128, dropout=0.2)
    )
  )

**Comment by ldulcic** [comment]
Hi @mymailpath, I'm not sure what is the question/problem here, could you be more specific?

**Comment by mymailpath** [comment]
No thanks. After some hours of debugging find an solution for my above question. :)

**Comment by mymailpath** [comment]
No thanks , next stage moved.
